segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -148,13 +144,16 @@ private void report() {
         private final OpStatsLogger processOperationsLatency;
         private final OpStatsLogger processOperationsBatchSize;
         private final String[] containerTag;
+        private final HashMap<String, String[]> throttlerTags = new HashMap<>();
```

Comment:
I'm not a big fan of this type of caching, for a couple of reasons:
1. You need to do a lookup in this map every time you report something.
2. The map's keys are not the same as the ones that are in `ThrottlerCalculator`. If we decide (for whatever reason) to rename, remove or add one in there, this will not work.
3. If (continued from 2 above), there is no entry in this map for a throttler name, then it will return null - how will the reporting work? will it throw a NullPtr? 

I am OK with generating a new array every time we invoke this. Generating a 4-element array is insignificant compared to all the other work that is done in the Operation Processor so there's no need to try to optimize it.